kubernetes/machine-class.yaml

```diff
@@ -33,6 +33,7 @@ providerSpec:
     storageProfile:
       imageReference:
         urn: sap:gardenlinux:greatest:184.0.0
+        # communityGalleryImageID: /CommunityGalleries/<community-gallery-id>/Images/<image-name>/Versions/<image-version>
```

Comment:
It would be better if `urn` and `communityGalleryImageID` are in line after commenting, will be easier to understand that they are on the same level.